libs/sdk-model/api/sdk-model.api.md

```diff
@@ -1180,6 +1180,19 @@ export interface IDrillToLegacyDashboard extends IDrill {
     type: "drillToLegacyDashboard";
 }
 
+// @public
+export interface IEntitlementDescriptor {
+    // (undocumented)
```

Comment:
Pls can you document this public interface props

Reply:
fixed